docs/HOWTO_add_routing_tiles.MD

```diff
@@ -0,0 +1,62 @@
+HOWTO_Add_routing tiles for Wahoo Elemnt ROAM
+
+Wahoo Elemnt ROAM is using OSM based map, and the base map tiles can be either downloaded from Wahoo update server (url: https://cdn.wahooligan.com/wahoo-maps/{version}/tiles/{zoom}/{x}/{y}.map.lzma; latest map version = 12, zoom level =8, x, y = map tile index/coordinates) or generated by using WahooMapsCreator (https://github.com/treee111/wahooMapsCreator). Besides the base map tiles, routing tiles are also required to support routing/navigation features (i.e. turn by turn support for Strava route). And it seems Valhalla (https://valhalla.readthedocs.io/en/latest/api/turn-by-turn/overview/) is used by Wahoo Elemnt ROAM as routing services. Following are the detailed instructions for getting routing tiles.
+
+1. Directory hierarchy in Wahoo Elelmnt ROAM device
+maps
+   | - routing
+   |      | - routing-config.json
+   |      | - 2
+   |           |- 000
+   |               | - xxx
+   |               |    |- yyy.gph.lzma
+   |               |    | -  ……
+   |               |    | - yyy.gph.lzma
+   |               | - ……
+   |               | - ……
+   |               | - ……
+   |               | - xxx
+   |               |    | - yyy.gph.lzma
+   |               |    | -  ……
+   |               |    | - yyy.gph.lzma
+   | - tiles
+	| - 8
+	    | - xxx
+	    |    | - yyy.map.lzma
+	    |    | -  ……
+	    |    | - yyy.map.lzma
+	    | - ……
+	    | - ……
+	    | - ……
+	    | - xxx
+	    |    | - yyy.map.lzma
+	    |    | -  ……
+	    |    | - yyy.map.lzma
+
+
+2. How to get routing tiles?
+	a. Option a: download from wahoo update server

```

Comment:
the indentation is not reflected in Github.
Lists could help: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists